src/webplugin/popup.css

```diff
@@ -0,0 +1,174 @@
+#openWalletButton {
```

Comment:
Seeing how the entire project uses Sass I think this file should be rewritten as Sass as well. This does require transpiling the Sass to CSS which is not automatically done by Ionic since it is 'outside' of the Ionic project. We might need to add the Sass compiler ourselves for this step. 

What do you think @DanielLeushuis ?